### PR TITLE
Ignore method on DataRequestParser process

### LINKED ISSUE #5: [ReflectionException] Property hasNotifications does not exist
On command 

`php app/console postman:collection:build -vvv`

I get an error : [ReflectionException] Property hasNotifications does not exist 

Context : I've an entity named 'reservation' with a method 'isHasNotifications' that is not linked to a property 'hasNotifications'

See exception trace below

```
Exception trace:
 () at /Users/cedric/Desktop/dev/erb-api/vendor/api-platform/postman-collection-generator/src/RequestParser/DataRequestParser.php:66
 ReflectionClass->getProperty() at /Users/cedric/Desktop/dev/erb-api/vendor/api-platform/postman-collection-generator/src/RequestParser/DataRequestParser.php:66
 PostmanGeneratorBundle\RequestParser\DataRequestParser->parse() at /Users/cedric/Desktop/dev/erb-api/vendor/api-platform/postman-collection-generator/src/RequestParser/RequestParserChain.php:38
 PostmanGeneratorBundle\RequestParser\RequestParserChain->parse() at /Users/cedric/Desktop/dev/erb-api/vendor/api-platform/postman-collection-generator/src/Generator/RequestGenerator.php:65
 PostmanGeneratorBundle\Generator\RequestGenerator->generate() at /Users/cedric/Desktop/dev/erb-api/vendor/api-platform/postman-collection-generator/src/Generator/CollectionGenerator.php:90
 PostmanGeneratorBundle\Generator\CollectionGenerator->generate() at /Users/cedric/Desktop/dev/erb-api/vendor/api-platform/postman-collection-generator/src/Command/PostmanCollectionBuildCommand.php:105
 PostmanGeneratorBundle\Command\PostmanCollectionBuildCommand->execute() at /Users/cedric/Desktop/dev/erb-api/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:256
 Symfony\Component\Console\Command\Command->run() at /Users/cedric/Desktop/dev/erb-api/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:807
 Symfony\Component\Console\Application->doRunCommand() at /Users/cedric/Desktop/dev/erb-api/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:186
 Symfony\Component\Console\Application->doRun() at /Users/cedric/Desktop/dev/erb-api/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:86
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /Users/cedric/Desktop/dev/erb-api/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:117
 Symfony\Component\Console\Application->run() at /Users/cedric/Desktop/dev/erb-api/app/console:29
```

### PR DESCRIPTION
**Pull request by vincentchalamon**
Fix #5

### CODE REVIEW

**Comment by vincentchalamon** [comment]
@dunglas Can you fix StyleCI configuration in this project please?

**Comment by vincentchalamon** [comment]
Ping @api-platform/core-team

File: src/RequestParser/DataRequestParser.php [hunk 1]
```diff
@@ -62,10 +62,15 @@ public function parse(Request $request)
...
-                'Symfony\Component\Serializer\Annotation\Groups'
-            );
+            if ($classMetadata->getReflectionClass()->hasProperty($attributeMetadata->getName())) {
+                $groups = $this->reader->getPropertyAnnotation(
+                    $classMetadata->getReflectionClass()->getProperty($attributeMetadata->getName()),
+                    'Symfony\Component\Serializer\Annotation\Groups'
+                );
+            } else {
+                // Attribute is not a property: ignore it
+                continue;
```
Comment (sroze) [inline src/RequestParser/DataRequestParser.php:67]: Can you put the `continue` in the `if` instead? That's far clearer IMO

Comment (vincentchalamon) [inline src/RequestParser/DataRequestParser.php:67]: Fixed

**Pull request merged**